core/src/apps/workflow_handlers.py

```diff
@@ -204,6 +204,12 @@ def _find_message_handler_module(msg_type: int) -> str:
         if msg_type == MessageType.SolanaSignTx:
             return "apps.solana.sign_tx"
 
+        # benchmark
```

Comment:
limit to `if __debug__` ?

Reply:
Done in https://github.com/trezor/trezor-firmware/pull/4101/commits/f032927e94cbda9c6ef559cab98317c35afb85b8.